tests/e2e/specs/shopper/filter-products-by-price.test.ts

```diff
@@ -59,7 +59,7 @@ const setMaxPrice = async () => {
 	await page.click( selectors.frontend.priceMaxAmount, {
 		clickCount: 3,
 	} );
-	await page.keyboard.type( '1.99' );
+	await page.keyboard.type( '2' );
```

Comment:
Haha that's interesting. Why this change?

Reply:
Just for debug reason, I restored the change.